NeoMathEngine/src/CPU/CpuMathEngineDnn3dConv.cpp

```diff
@@ -331,16 +333,15 @@ void CCpuMathEngine::blob3dConvolution( const CCommon3dConvolutionDesc& desc, co
 
 				if( freeTermData != 0 ) {
 					addVectorToMatrixRows( outputTemp, outputTemp, outputTempGeometricalSize, result.Channels(),
-						result.Channels(), result.Channels(), *freeTermData );
+						result.Channels(), result.Channels(), GetRaw( *freeTermData ) );
```

Comment:
`GetRaw` can be removed by using `const float* freeTermDataPtr = freeTermData == nullptr ? nullptr : GetRaw( *freeTermData )` initialized outside of `for( int b = batchStart ...`